package.json

```diff
@@ -15,15 +21,16 @@
   "collection:main": "dist/collection/index.js",
   "unpkg": "dist/mui-core/mui-core.js",
   "files": [
-    "dist/",
-    "loader/"
+    "dist/"
   ],
```

Comment:
`package.json.files` is the opposite of `.npmignore`. `files` starts with npm publishing nothing, except for what’s in here. `.npmignore` starts with npm publishing everything, except for the items excluded.

Yes it’s unclear how they interact, but IMO `.npmignore` is clearer in purpose. I’d just delete `"files"` if you can. Or if Stencil yells at you, keep it but put `"loader/"` back to make sure that it always publishes to npm too